spec/system/user_page/user_preferences_security_spec.rb

```diff
@@ -36,6 +36,11 @@
 
       user_menu.sign_out
 
+      # puts <<~STRING
+      # public_key_base64 = \"#{user.second_factor_security_keys.first.public_key}\"
+      # private_key_string = \"#{authenticator.credentials.first.private_key}\"
+      # STRING
+
```

Comment:
I guess we can get rid of this now? 🙂

Reply:
I'm keeping this around for https://github.com/discourse/discourse/pull/32459/files#diff-5606dc05da0431ea31ad3c80926d8cb030431a250ede4944cbfce301fecb802fR234-R235 as I couldn't think of a better idea.